src/store/middlewares/notification.js

```diff
@@ -0,0 +1,23 @@
+import actionTypes from '../../constants/actions';
+import Notification from '../../utils/notification';
+
+const notificationMiddleware = (store) => {
+  const notify = Notification.init();
+  return next => (action) => {
+    next(action);
+
+    const { account } = store.getState();
```

Comment:
oh... you right. i handled notification with hard-coded value for testing after your first comment.